grasping_poses:
- id:               stone_9_actual
  index:             1
  pose:
    position:
      x:            0.00716126
      y:            0.00543468
      z:            -0.212052
    orientation:
      x:            -0.0298774
      y:            0.0119403
      z:            0.830293
      w:            0.556397
  finger_pos:
   -            44.910 #id: 2
   -            44.590 #id: 5
   -            22.710 #id: 8
   -            5.540 #id: 1
   -            70.280 #id: 4
   -            7.400 #id: 7
   -            39.280 #id: 0
   -            -68.990 #id: 3
   -            -27.940 #id: 6

# 90 deg rotated
- id:               stone_9_actual
  index:             3
  pose:
    position:
      x:            -0.0479336
      y:            -0.182969
      z:            -0.00168391
    orientation:
      x:            0.179071
      y:            0.689527
      z:            0.696947
      w:            -0.0821662
  finger_pos:
   -            63.510 #id: 2
   -            66.910 #id: 5
   -            72.250 #id: 8
   -            -0.680 #id: 1
   -            54.650 #id: 4
   -            -2.520 #id: 7
   -            44.620 #id: 0
   -            -2.120 #id: 3
   -            -20.030 #id: 6


 #################################################
- id:               stone_8_actual
  index:             1
  pose:
    position:
      x:            0.0139307
      y:            0.0753583
      z:            -0.190
    orientation:
      x:            0.0712109
      y:            -0.1499
      z:            0.886836
      w:            0.431254
  finger_pos:
   -            -29.000 #id: 2
   -            41.680 #id: 5
   -            35.360 #id: 8
   -            28.100 #id: 1
   -            70.740 #id: 4
   -            -7.700 #id: 7
   -            32.630 #id: 0
   -            -94.130 #id: 3
   -            -12.830 #id: 6



#180 rotated around x
- id:               stone_8_actual
  index:             3
  pose:
    position:
      x:            0.0450622
      y:            0.0574769
      z:            -0.198124
    orientation:
      x:            0.161886
      y:            0.0553057
      z:            -0.405666
      w:            0.897869
  finger_pos:
   -            44.140 #id: 2
   -            26.760 #id: 5
   -            26.880 #id: 8
   -            -19.590 #id: 1
   -            70.740 #id: 4
   -            13.350 #id: 7
   -            28.670 #id: 0
   -            99.840 #id: 3
   -            -28.030 #id: 6


 #################################################
- id:               stone_2_actual
  index:             9
  pose:
    position:
      x:            0.0740414
      y:            -0.0123772
      z:            0.185039
    orientation:
      x:            0.921411
      y:            0.362674
      z:            -0.097308
      w:            -0.0999994
  finger_pos:
   -            35.150 #id: 2
   -            21.750 #id: 5
   -            34.750 #id: 8
   -            3.630 #id: 1
   -            70.740 #id: 4
   -            7.000 #id: 7
   -            36.780 #id: 0
   -            99.840 #id: 3
   -            -25.220 #id: 6



#z:180deg yellow tape side
- id:               stone_7_actual
  index:             3
  pose:
    position:
      x:            0.0609668
      y:            -0.0451621
      z:            -0.188
    orientation:
      x:            -0.16296
      y:            0.135397
      z:            0.952165
      w:            -0.220213
  finger_pos:
   -            38.750 #id: 2
   -            57.910 #id: 5
   -            36.790 #id: 8
   -            -4.270 #id: 1
   -            66.090 #id: 4
   -            8.680 #id: 7
   -            30.300 #id: 0
   -            99.400 #id: 3
   -            -21.440 #id: 6

#################################################
#two finger
- id:               stone_2_actual
  index:             4
  pose:
    position:
      x:            0.0361924
      y:            -0.00235574
      z:            0.186404
    orientation:
      x:            0.81302
      y:            0.575375
      z:            -0.00727015
      w:            -0.0888225
  finger_pos:
   -            55.540 #id: 2
   -            -40.250 #id: 5
   -            47.700 #id: 8
   -            -1.190 #id: 1
   -            -15.750 #id: 4
   -            6.600 #id: 7
   -            40.640 #id: 0
   -            29.170 #id: 3
   -            -22.500 #id: 6



#180 rotated around z
- id:               stone_2_actual
  index:             11
  pose:
    position:
      x:            -0.0379952
      y:            -0.0563526
      z:            -0.197227
    orientation:
      x:            -0.119721
      y:            0.0512476
      z:            -0.512187
      w:            0.848944
  finger_pos:
   -            -8.300 #id: 2
   -            18.650 #id: 5
   -            -20.120 #id: 8
   -            27.480 #id: 1
   -            60.550 #id: 4
   -            28.470 #id: 7
   -            44.280 #id: 0
   -            99.930 #id: 3
   -            -11.250 #id: 6


#################################################
- id:               stone_4_actual
  index:             2
  pose:
    position:
      x:            0.0257916
      y:            -0.0372826
      z:            -0.205448
    orientation:
      x:            -0.0785828
      y:            -0.0238043
      z:            0.150529
      w:            0.98519
  finger_pos:
   -            -37.210 #id: 2
   -            75.090 #id: 5
   -            -39.640 #id: 8
   -            34.270 #id: 1
   -            52.010 #id: 4
   -            28.210 #id: 7
   -            36.600 #id: 0
   -            -82.520 #id: 3
   -            -15.380 #id: 6


###################################################
- id:               stone_5_actual
  index:             2
  pose:
    position:
      x:            0.186812
      y:            -0.0546814
      z:            0.0517831
    orientation:
      x:            0.673219
      y:            -0.436768
      z:            -0.560281
      w:            0.20517
  finger_pos:
   -            19.510 #id: 2
   -            50.530 #id: 5
   -            -13.560 #id: 8
   -            5.760 #id: 1
   -            64.480 #id: 4
   -            22.230 #id: 7
   -            39.700 #id: 0
   -            94.440 #id: 3
   -            -27.770 #id: 6



# 180 rot around z
- id:               stone_5_actual
  index:             1
  pose:
    position:
      x:            0.191219
      y:            -0.0390066
      z:            -0.0214041
    orientation:
      x:            -0.654159
      y:            0.191636
      z:            0.723329
      w:            -0.110211
  finger_pos:
   -            56.770 #id: 2
   -            30.200 #id: 5
   -            2.040 #id: 8
   -            -12.320 #id: 1
   -            8.720 #id: 4
   -            28.560 #id: 7
   -            9.400 #id: 0
   -            39.550 #id: 3
   -            -10.190 #id: 6
